content/id/docs/concepts/scheduling/scheduling-framework.md

```diff
@@ -0,0 +1,249 @@
+
+_Plugin_ ini digunakan untuk memodifikasi skor sebelum penjadwal menghitung 
+peringkat akhir Node-Node. _Plugin_ yang mendaftar untuk titik ekstensi ini akan 
+dipanggil dengan hasil [Score](#score) dari _plugin_ yang sama. Hal ini dilakukan
+sekali untuk setiap _plugin_ dan setiap siklus penjadwalan.
+
+Sebagai contoh, anggaplah sebuah _plugin_ `BlinkingLightScorer` memberi peringkat 
+pada Node-Node berdasarkan berapa banyak kedipan lampu yang mereka miliki.
+
+```go
+func ScoreNode(_ *v1.pod, n *v1.Node) (int, error) {
+    return getBlinkingLightCount(n)
+}
+```
+
+Namun, jumlah maksimum kedipan lampu mungkin kecil jika dibandingkan dengan
+`NodeScoreMax`. Untuk memperbaikinya, `BlinkingLightScorer` juga harus mendaftar
+untuk titik ekstensi ini.
+
+```go
+func NormalizeScores(scores map[string]int) {
+    highest := 0
+    for _, score := range scores {
+        highest = max(highest, score)
+    }
+    for node, score := range scores {
+        scores[node] = score*NodeScoreMax/highest
+    }
+}
+```
+
+Jika ada _plugin_ NormalizeScore yang menghasilkan hasil yang salah, maka siklus 
+penjadwalan dibatalkan.
+
+{{< note >}}
+_Plugin_ yang ingin melakukan pekerjaan "pra-pemesanan" harus menggunakan
+titik ekstensi NormalizeScore.
+{{< /note >}}
+
+### Reserve
+
+Ini adalah titik ekstensi yang bersifat informasi. _Plugin_ yang mempertahankan
+keadaan _runtime_ (alias "_stateful plugins_") harus menggunakan titik ekstensi ini
+untuk diberitahukan oleh penjadwal ketika sumber daya pada suatu Node dicadangkan
+untuk Pod yang telah disiapkan. Proses ini terjadi sebelum penjadwal benar-benar
+mengikat Pod ke Node, dan itu ada untuk mencegah kondisi balapan (_race conditions_)
+ketika penjadwal menunggu agar pengikatan berhasil.
+
+Ini adalah langkah terakhir dalam siklus penjadwalan. Setelah Pod berada dalam 
+status dicadangkan, maka itu akan memicu _plugin_ [Unreserve](#unreserve)
+(apabila gagal) atau _plugin_ [PostBind](#post-bind) (apabila sukses) 
+di akhir siklus pengikatan.
+
+### Permit
+
+_Plugin_ Permit dipanggil pada akhir siklus penjadwalan untuk setiap Pod
+untuk mencegah atau menunda pengikatan ke Node kandidat. _Plugin_ Permit dapat 
+melakukan salah satu dari ketiga hal ini:
+
+1. ** approve ** \
+     Setelah semua _plugin_ Permit menyetujui sebuah Pod, Pod tersebut akan dikirimkan untuk diikat.
+
+2. ** deny ** \
+     Jika ada _plugin_ Permit yang menolak sebuah Pod, Pod tersebut akan dikembalikan ke 
+     antrian penjadwalan. Hal ini akan memicu _plugin_ [Unreserve](#unreserve).
+
+3. ** wait ** (dengan batas waktu) \
+     Jika _plugin_ Permit menghasilkan "wait", maka Pod disimpan dalam
+     daftar Pod "yang menunggu" internal, dan siklus pengikatan Pod ini dimulai tetapi akan langsung diblokir
+     sampai mendapatkan [_approved_](#frameworkhandle). Jika waktu tunggu habis, ** wait ** menjadi ** deny **
+     dan Pod dikembalikan ke antrian penjadwalan, yang memicu _plugin_ [Unreserve](#unreserve).
```

Comment:
![image](https://user-images.githubusercontent.com/2609951/79129808-0a6ff380-7dd0-11ea-8398-446cfedc43f6.png)

same as above, ut's rendered as plain text, should be bold